### DISCUSSION TITLE
How to remove i18n ?

### DISCUSSION THREAD

**Discussion by superhelten** [UnlyEd/next-right-now#386] [Q&A]

**Comment by Vadorequest** [comment]
Well, the documentation lists the libraries that are being used for i18n, but you'll need to remove all usages of functions from those libs and all imports, too.

Functions like `t` (means "translate") and such will need to be removed everywhere in the codebase.

I can't really guide you more than that, there are too many use-cases. But if you don't understand why some particular piece of code doesn't compile, please post a screenshot and I'll give you some pointers.

**Reply by Vadorequest** [reply]
@spaciy So, I made a video to explain things, I believe it'll be easier to understand.

https://watch.screencastify.com/v/AV3oEMYCGN6rAyP28Cxe

Here is a Git patch you can apply on your project, which will disable all call to fetching translations from Locize provider, and will mock all existing calls to the `t` function. (means "translate") 

_Note that the static-i18n demo page example doesn't compile anymore for some reason I haven't found out, but it shouldn't be an issue considering you're gonna get rid of it anyway. Other pages works properly._

[WIP_not_working_on__static-i18n_page.patch.zip](https://github.com/UnlyEd/next-right-now/files/7617752/WIP_not_working_on__static-i18n_page.patch.zip)
